blockchain/chain_sync/src/tipset_syncer.rs

```diff
@@ -1297,43 +1227,33 @@ async fn validate_block<
         .map_err(|e| (*block_cid, e.into()))?;
 
     // Async validations
+    let validations = FuturesUnordered::new();
 
     // Check block messages
     let v_block = Arc::clone(&block);
     let v_base_tipset = Arc::clone(&base_tipset);
     let v_state_manager = Arc::clone(&state_manager);
     validations.push(task::spawn_blocking(move || {
-        check_block_messages::<_, V>(v_state_manager, &v_block, &v_base_tipset)
-            .map_err(|e| TipsetRangeSyncerError::Validation(e.to_string()))
-    }));
-
-    // Miner validations
-    let v_state_manager = Arc::clone(&state_manager);
-    let v_block = Arc::clone(&block);
-    let v_base_tipset = Arc::clone(&base_tipset);
-    validations.push(task::spawn_blocking(move || {
-        let headers = v_block.header();
-        validate_miner(
-            &v_state_manager,
-            headers.miner_address(),
-            v_base_tipset.parent_state(),
-        )
+        check_block_messages::<_, C>(v_state_manager, &v_block, &v_base_tipset)
     }));
 
     // Base fee check
+    let smoke_height = state_manager.chain_config.epoch(Height::Smoke);
     let v_base_tipset = Arc::clone(&base_tipset);
     let v_block_store = state_manager.blockstore_cloned();
     let v_block = Arc::clone(&block);
     validations.push(task::spawn_blocking(move || {
-        let base_fee =
-            chain::compute_base_fee(v_block_store.as_ref(), &v_base_tipset, smoke_height).map_err(
-                |e| {
-                    TipsetRangeSyncerError::Validation(format!("Could not compute base fee: {}", e))
-                },
-            )?;
+        let base_fee = chain::compute_base_fee(
+            v_block_store.as_ref(),
+            &v_base_tipset,
+            smoke_height,
+        )
+        .map_err(|e| {
+            TipsetRangeSyncerError::<C>::Validation(format!("Could not compute base fee: {}", e))
```

Comment:
```suggestion
            TipsetRangeSyncerError::<C>::Validation(format!("Could not compute base fee: {e}"))
```

Reply:
It's a new addition to Rust (since 1.58). https://www.rustnote.com/blog/format_strings.html

Definitely not a big deal though. :)